src/go/rpk/pkg/cli/cluster/partitions/status.go

```diff
@@ -95,16 +95,18 @@ investigation. A few areas to investigate:
 }
 
 func printBalancerStatus(pbs adminapi.PartitionBalancerStatus) {
-	const format = `Status:                       %v
-Seconds Since Last Tick:      %v
-Current Reassignment Count:   %v
+	const format = `Status:                               %v
+Seconds Since Last Tick:              %v
+Current Reassignment Count:           %v
 `
 	fmt.Printf(format, pbs.Status, pbs.SecondsSinceLastTick, pbs.CurrentReassignmentsCount)
-
+	if pbs.PartitionsPendingForceRecovery != nil && pbs.PartitionsPendingRecoveryList != nil {
+		fmt.Printf("Partitions Pending Recovery (%v):      %v\n", *pbs.PartitionsPendingForceRecovery, pbs.PartitionsPendingRecoveryList)
```

Comment:
We should create a follow up issue to make this formatting easier. The only difference between this and the out column formatter is the column formatter capitalizes the left column and it only has two spaces. I think we can make that configurable.

Reply:
This is already possible with our table writer. We can just create a table without sections and it works. :partying_face: 

I just changed the code to use that, it's way easier to follow now. Thanks for pointing this out.